script/tool/lib/src/firebase_test_lab_command.dart

```diff
@@ -59,7 +59,7 @@ class FirebaseTestLabCommand extends PackageLoopingCommand {
         help:
             'Device model(s) to test. See https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run for more info');
     argParser.addOption('results-bucket',
-        defaultsTo: 'gs://flutter_firebase_testlab');
+        defaultsTo: 'gs://flutter_firebase_testlab_staging');
```

Comment:
This is the bucket only, I assume there is another CL on recipes to change the GCP project used by `gcloud firebaselab ...` command. Am I right?

Reply:
Yes, my other PR in the recipes repo is changing the project to flutter-infra-staging. It is changing the recipe for the firebase commands.